### PR TITLE
Add robot_email attr to container push bzl rules

### PR DESCRIPTION
**Pull request by tabletenniser**
The robot_email attr is used at Google internally and github users should not be using it.

### CODE REVIEW

File: container/push.bzl [hunk 1]
```diff
@@ -187,6 +187,11 @@ container_push = rule(
             default = Label("//container:push-tag.sh.tpl"),
             allow_single_file = True,
         ),
+        "robot_email": attr.string(
+            default = "",
```
Comment (nlopezgi) [inline container/push.bzl]: Move to line 159

Comment (tabletenniser) [reply container/push.bzl]: Created a new commit to move it, both here and contrib/push-all.bzl.

**Pull request merged**